http/1.6/modules/ROOT/pages/index.adoc

```diff
@@ -99,23 +77,19 @@ The next example uses the incoming HTTP request query parameters and customizes
 </flow>
 ----
 
-Considering the example above, now calling `+http://localhost:8081/greet?name=Ana+`
-returns `Hello, Ana! I am an HTTP server.`, while calling `+http://localhost:8081/greet+`
-returns `Hello, world! I am an HTTP server.`.
+If the HTTP *Listener* source is triggered with `+http://localhost:8081/greet?name=Ana+`, the connector
+returns `Hello, Ana! I am an HTTP server.`.  Although, when the source is triggered with `+http://localhost:8081/greet+`, the connector returns `Hello, world! I am an HTTP server.`.
 
-You can find a complete overview of the HTTP listener features in the xref:http-listener-ref.adoc[HTTP Listener Configuration Reference].
+To get further information on the HTTP *Listener* source, refer to the xref:http-listener-ref.adoc[Configure an HTTP Listener source] documentation.
 
 == HTTP Clients
 
-HTTP clients enable you to send requests to HTTP services and handle their responses. For
-the connector, this means executing a request and returning the answer. This is
-handled by the HTTP request, an operation that the connector provides.
+HTTP clients enable you to send requests to HTTP services and manage their responses. The HTTP *Request* operation executes HTTP requests and returns the answer.
```

Comment:
Tense agreement > "their responses" matches with "answers." Maybe say "returns the corresponding answers."